CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 3.1.1
+  - Fix: avoid reusing per-identity codec instances for differing identities. Removes a very minor optimization so that stateful codecs like CSV can work reliably. 
+
```

Comment:
Please add the PR link to the changelog so that it'll be picked up in release notes:

[#70](https://github.com/logstash-plugins/logstash-codec-multiline/pull/70).

Reply:
Resolved in 2a40c2165ffb0570f1c0fbdc90946e02254f18ac